docs/guide/macOS.md

```diff
@@ -1,27 +1,27 @@
 # macOS Virtual Machines
 
-It is possible to run macOS Virtual Machines the same way one can run [Linux containers](linux.md) on macOS Community Cluster. 
+It is possible to run macOS Virtual Machines the same way one can run [Linux containers](linux) on macOS Community Cluster. 
```

Comment:
`.md` links are OK. There is some shenanigans with non-relative links though. See #145

Reply:
Okay

Reply:
fixed